miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -517,8 +517,8 @@ private static SampleDto copySampleFields(@Nonnull Sample from, @Nonnull SampleD
     dto.setAlias(from.getAlias());
     dto.setProjectId(from.getProject().getId());
     dto.setProjectName(from.getProject().getName());
-    dto.setProjectAlias(from.getProject().getAlias());
-    dto.setProjectShortName(from.getProject().getShortName());
+    dto.setProjectAlias(from.getProject().getTitle());
+    dto.setProjectShortName(from.getProject().getCode());
```

Comment:
Please modify `SampleDto` to change `get/setProjectAlias` -> `get/setProjectTitle` and `get/setProjectShortName` -> `get/setProjectCode`